### PR TITLE
Ah shit.

### PR DESCRIPTION
**Pull request by JamieD1**
Ah shit.

### CODE REVIEW

**Review by redmoogle — APPROVED** [review]
Good one Jamie

**Comment by patpol4** [comment]
Haha code, am i right

**Review by jcatjmeow — APPROVED** [review]
Smh

**Review by patpol4 — APPROVED** [review]
great mapping

**Comment by MrHorizons** [comment]
Here we go again.

**Comment by patpol4** [comment]
lol

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -1266,20 +1266,7 @@ Traitors and the like can also be revived with the previous role mostly intact.
...
-				return
-			for(var/turf/T in centcomarea)
-				if(T.density)
-					continue
-				turfs.Add(T)
-	var/turf/T = safepick(turfs)
-	if(!T)
-		to_chat(src, "Nowhere to jump to!", confidential=TRUE)
-		return
+	var/turf/T = locate(196,82,1)
```
Comment (patpol4) [inline code/modules/admin/verbs/randomverbs.dm:1269]: Can't wait until we get a CC change and this has to be changed

**Comment by Bibby0110** [comment]
Jamie what the fuck is this, if you merge this you’re clinically insane

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (patpol4) [reply code/modules/admin/verbs/randomverbs.dm:1269]: wouldnt it be better if you make a landmark and place it on the map

**Pull request merged**